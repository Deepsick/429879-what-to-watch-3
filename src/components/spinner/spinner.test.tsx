import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Spinner from './spinner';

it(`Should render Spinner component correctly`, () => {
  const node = renderer.create(
      <Spinner />
  ).toJSON();

  expect(node).toMatchSnapshot();
});
