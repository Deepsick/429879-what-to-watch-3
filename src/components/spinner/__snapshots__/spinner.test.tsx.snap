// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Spinner component correctly 1`] = `
Array [
  <div
    className="loader"
  />,
  <div
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    Loading...
  </div>,
]
`;
